### PR TITLE
Release automation

### PR DESCRIPTION
**Pull request by Akiyamka**

### CODE REVIEW

**Comment by IlyaIzr** [comment]
This script should create release branch when you're in main?

**Comment by Akiyamka** [comment]
> This script should create release branch when you're in main?

This script allow you create release branch by `npm run release`  command only when
- you on main
- main is sync with remote
- no unmerged releases right now  

also it:
- create branch
- increase version in package.lock and package.js
- push branch

**Review by IlyaIzr — APPROVED** [review]
I'd suggest to put script description and rules of usage to the script as comment or to Readme

File: scripts/auto-release/index.mjs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ * This script allow you create release branch by npm run release command only when
+ * • you on main
+ * • main is sync with remote
+ * • no unmerged releases right now
+ * also it:
+ * • create branch
+ * • increase version in package.lock and package.js
+ * • create commit
+ * • push branch
+ */
```
Comment (IlyaIzr) [inline scripts/auto-release/index.mjs]:
```suggestion
 * This script allows you to create release branch by npm run release command only when
 * • you're on main
 * • main is sync with remote
 * • no unmerged releases right now
 * also it:
 * • creates branch
 * • increases version in package.lock and package.json
 * • creates commit
 * • pushes branch
 */
```

**Pull request merged**